openshift/keycloak-db.app.yaml

```diff
@@ -31,7 +31,13 @@ objects:
       type: Recreate
       recreateParams:
         timeoutSeconds: 600
-      resources: {}
+      resources:
+        requests:
+          cpu: 50m
+          memory: 100Mi
+        limits:
+          cpu: 500m
```

Comment:
Here we have a problem. With a single pod atm, KC can be quite demanding in terms of cpu and memory. e.g. 
![image](https://cloud.githubusercontent.com/assets/3602792/25272409/f57aa7ec-2687-11e7-9e71-1c8bd35d364b.png)